frontend/src/visualization/filtering/ShortestPathNodeSelection.tsx

```diff
@@ -6,14 +6,17 @@ import useService from '../../dependency-injection/useService';
 import QueryResultStore from '../../stores/QueryResultStore';
 import useObservable from '../../utils/useObservable';
 
+type nodeChangedCallback = (id: number | null) => void;
```

Comment:
use capital letter for the first letter since this is a custom type

Reply:
Done!